migrations/2023-06-23-172550_expire_password_reset_tokens/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table password_reset_request add column expires_at timestamp not null;
```

Comment:
Adding a not null column without a default means that any existing password_reset_request rows will result in a broken migration. I suggest either deleting them all before running this, or adding a default.